test-requirements.txt

```diff
@@ -2,7 +2,7 @@ pytest==3.0.6
 mock
 -e git+https://github.com/uc-cdis/cdisutils-test.git@0.2.2#egg=cdisutilstest
 -e git+https://github.com/uc-cdis/cdis-python-utils.git@0.1.7#egg=cdispyutils
--e git+https://github.com/uc-cdis/indexd.git@0.7.0#egg=indexd
+-e git+https://github.com/uc-cdis/indexd.git@63f6874ae9dfe9eea0e9525575b58a368b96f49d#egg=indexd
```

Comment:
wait till indexd has a tag for the changes.

Reply:
created 0.8.0